rm/rm_foms.c

```diff
@@ -585,6 +586,11 @@ static int request_pre_process(struct m0_fom *fom,
 			m0_rm_incoming_fini(&rfom->rf_in.ri_incoming);
 			goto err;
 		}
+		rfom->rf_in.ri_incoming.rin_remote = debtor;
+		cnt = m0_ref_read(&debtor->rem_incomings);
+		M0_LOG(M0_DEBUG, "rm_remote=%p incomings: %d -> %d",
+		       debtor, cnt, cnt + 1);
+		m0_ref_get(&debtor->rem_incomings);
```

Comment:
Please introduce helper functions loan_{get,put}() and use them.

Reply:
Will do, thanks.

Reply:
Ah, now I recall why I didn't do it in the 1st place: I wanted the calling function to be visible in the log, not loan_{get,put}() (which would not be of very much use). Will do it with a macro...